app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> None:
+        return round((
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * self.average_rating
+            / self.distance_from_city_center), 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+        return car
+
+    def rate_service(self, rating: int) -> None:
+        self.average_rating = round(
+            (self.average_rating * self.count_of_ratings + rating)
+            / (self.count_of_ratings + 1), 1)
+        self.count_of_ratings += 1
+
+    def serve_cars(self, cars: list[Car]) -> None:
+        money = 0.0
+        for car in cars:
+            price = self.calculate_washing_price(car=car)
+            car = self.wash_single_car(car=car)
+            if car.clean_mark == self.clean_power:
+                money += price
```

Comment:
```suggestion
            if car.clean_mark < self.clean_power:
                money += self.calculate_washing_price(car)
                self.wash_single_car(car)
```
Due to task you must check this condition